pkg/scheduler/api/node_info.go

```diff
@@ -214,8 +214,9 @@ func (ni *NodeInfo) RemoveTask(ti *TaskInfo) error {
 
 	task, found := ni.Tasks[key]
 	if !found {
-		return fmt.Errorf("failed to find task <%v/%v> on host <%v>",
+		glog.Warningf("failed to find task <%v/%v> on host <%v>",
```

Comment:
Why change that? It should be error here.

Reply:
Here is hzxuzhonghu's comments. 
> As a following, i prefer removing the return error from `NodeInfo.RemoveTask`. If the task not exists, it must be something wrong else where. Just warning here is enough.

Reply:
Yep, removing a non existing task does not make harm to the scheduler.

Reply:
@k82cn 

Should I need to change back to Errorf again? If no need, please help to approve. Thanks.